### PR TITLE
WIP: created a new page

### PR DESCRIPTION
**Pull request by s-abdullaev**
please review my code

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (Bachelor052) [inline index.html:1]: OKay

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Welcome Page</title>
+</head>
+<body>
+	<h1>Welcome Page</h1>
+	<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Voluptas, ab tempore quia autem praesentium dolorem dolore, veritatis numquam maiores debitis odio ullam, doloremque omnis dignissimos assumenda ratione repudiandae, dolorum cum.</p>
```
Comment (Bachelor052) [inline index.html:18]: Change it to a meaningful text.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Welcome Page</title>
+</head>
+<body>
```
Comment (Bachelor052) [inline index.html:7]: Add navigation bar

**Review by Bachelor052 — CHANGES_REQUESTED** [review]
Do it

File: index.html (hunk 2 above)
Comment (s-abdullaev) [reply index.html:18]: THis text is just a temporary text. We will change it later

Comment (Bachelor052) [reply index.html:18]: OKay then

File: index.html (hunk 3 above)
Comment (s-abdullaev) [reply index.html:7]: added in d0cbfd3b4b42255815c95cb7ca3b90fb8bbb7044

Comment (Bachelor052) [reply index.html:7]: Wow. Wonderful

**Pull request merged**